### DISCUSSION TITLE
Completion menu collapses right after tab key up

### DISCUSSION THREAD

**Discussion by tony-sol** [marlonrichert/zsh-autocomplete#465] [Q&A] [answered]
After input any command appears suggestions for command and by pressing tab it expands to show more, but as tab key up, it collapsed again.
Is there ability to expand "more" list permanently?
Here is a bref explanation of what i mean:

https://user-images.githubusercontent.com/33671815/183674549-df0ef113-29cc-468e-b9fc-05c1418bfdd7.mov

Every time i press tab, list expands and immediately wrap again.
Is it bug or feature?

**Comment by marlonrichert** [comment, marked as answer]
> Is there ability to expand "more" list permanently?

Press <kbd>Down Arrow</kbd> or <kbd>Ctrl</kbd><kbd>Space</kbd>.

> Is it bug or feature?

It depends.™ What settings do you use with Autocomplete? It does not behave like this out of the box.

**Reply by tony-sol** [reply]
It's same with `Down Arrow` as with `Tab` :c
But yes, after some system shortcuts re-assignments, it's working with `Ctrl+Space`, thank you!
